cstudio7/bookapp
# #177829078 Server and Database Setup

#### What does this PR do?
Initialized Database and Server

#### Description of Task to be completed?
GET http://localhost:3000
Eslint Intialized
Balel Initialized
Sequelized Model and Migration file Initialized 


#### How should this be manually tested?
- After cloning the repo, cd into it and run npm or yarn install
- Initialize the app RUN npm run dev
- Using Postman verify the the endpoint http://localhost:3000 to verify on a GET request

#### What are the relevant pivotal tracker stories?
[Start #177829078]
